hangups/http_utils.py

```diff
@@ -46,6 +46,9 @@ def fetch(method, url, params=None, headers=None, cookies=None, data=None,
         else:
             error_msg = None
             break
+        finally:
+            if 'res' in locals():
+                res.close()
```

Comment:
Nice tester. I got some different results (converted to Python 3.4 syntax):
https://gist.github.com/tdryer/6460d4580a8270df18b08381b2ae0b74

Should we be calling `release` instead of `close` after successful requests so the connection stays open to allow keep-alive?